components/molecules/ENSInputField/ENSInputField.tsx

```diff
@@ -0,0 +1,44 @@
+import { CSS } from "../../../stitches.config";
+import { Text, UnstyledTextField, Flex } from "../../atoms";
+
+export interface ENSInputFieldProps {
+  css: CSS;
+  ensName: string;
+  value: string;
+  handleOnChange: (value: string) => void;
+  nonEditable?: boolean;
+  children?: React.ReactNode;
+}
+
+export const ENSInputField: React.FC<ENSInputFieldProps> = ({
+  children,
+  ensName,
+  value,
+  handleOnChange,
+  nonEditable,
+  css,
+}): JSX.Element => (
+  <Flex css={css}>
+    <Flex css={{ gap: "$xxs" }}>
+      <Flex direction="column">
+        {ensName && (
+          <Text size="4" color="textLight">
+            {ensName}
+          </Text>
+        )}
+        <UnstyledTextField
+          css={{
+            color: "$obolGreen",
+          }}
+          placeholder="Ethereum address or ENS name"
+          onChange={(e: { target: { value: string } }) =>
```

Comment:
It would be better to use the actual type of the event, `React.ChangeEvent<HTMLInputElement>`

Reply:
We call field.onChange when this is triggered, so we're passing a changeHandler from launchpad.